### PR TITLE
Cron jobs #24

### PR DESCRIPTION
**Pull request by timozattol**
* Create 2 new artisan commands: syncstudents & sendnotificationmails
 * syncstudents syncs the student list in the DB with the LDAP
 * sendnotificationmails sends emails on newly added ads to students
* Create scheduler entries to trigger those tasks
* Add laravel scheduler cron job into provision file
* Modify basic notification email view
* Create LDAP class to encapsulate LDAP operations

### CODE REVIEW

File: app/Agepinfo/LDAP.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+//==========================================================================
+//! EPFL LDAP Wrapper
+//  Provides a way to easily retrieve students from EPFL LDAP
+//  LDAP can only be reached on the EPFL campus or through VPN connection
+//==========================================================================
+
+final class LDAP {
+	const URL = 'ldaps://ldap.epfl.ch';
+	const PORT = 636;
+	const BASE = 'ou=etu,o=epfl,c=ch';
```
Comment (zifeo) [inline app/Agepinfo/LDAP.php]: Maybe create a ldap config file?

**Comment by zifeo** [comment]
Conflicts need to be resolved, but excellent work :+1:

File: app/Agepinfo/LDAP.php (hunk 1 above)
Comment (timozattol) [inline app/Agepinfo/LDAP.php]: Yep :+1:

Comment (timozattol) [inline app/Agepinfo/LDAP.php]: Done :heart:

**Comment by zifeo** [comment]
:+1:

**Pull request merged**